avocado/core/safeloader.py

```diff
@@ -89,7 +89,10 @@ def is_docstring_directive_enable(docstring):
     :rtype: bool
     """
     result = get_docstring_directive(docstring)
-    return result == 'enable'
+    if 'enable' in result:
```

Comment:
Well, as you don't mind about the functions names, let me propose something I believe is more efficient.